lib/ssh_scan/api.rb

```diff
@@ -103,10 +110,25 @@ class API < Sinatra::Base
         options[:sockets] <<
           "#{params[:target]}:#{params[:port] ? params[:port] : "22"}"
         options[:policy_file] = options[:policy]
+        options[:force] = params[:force] ? params[:force] : false
+        unless options[:force] == 'true'
+          available_result = settings.db.fetch_available_result(params)
+          unless available_result.nil?
+            if (Time.now -
+                Time.parse(available_result[:scanned_on].to_s)
+               ) / (60 * 60 * 24) < 1
+              return {
+                uuid: available_result[:uuid],
+                scanned_on: available_result[:scanned_on]
+              }.to_json
+            end
+          end
+        end
         options[:uuid] = SecureRandom.uuid
         settings.job_queue.add(options)
         {
-          uuid: options[:uuid]
+          uuid: options[:uuid],
+          scanned_on: Time.now
```

Comment:
This isn't really accurate (it's merely just tasked), also the scan result contains start and stop times for actual network i/o style scan.